lib/game_completion.rb

```diff
@@ -0,0 +1,19 @@
+module TicTacToe
+  class GameCompletion 
+
+    def initialize(console_ui) 
+      @console_ui = console_ui
+    end
+
+    def game_over_messages(game_result, board_array)
+      @console_ui.display_board(board_array)
+      @console_ui.puts_space
+      if game_result == "player_one" || game_result == "player_two"
```

Comment:
Since this is checking if the game has been won, why not just ask the GameEval class instead of re-implementing that logic here?